docs/src/pages/getting-started/page-layout-examples/sign-in/SignIn.js

```diff
@@ -17,31 +17,31 @@ const styles = theme => ({
   main: {
     width: 'auto',
     display: 'block', // Fix IE 11 issue.
-    marginLeft: theme.spacing.unit * 3,
-    marginRight: theme.spacing.unit * 3,
-    [theme.breakpoints.up(400 + theme.spacing.unit * 3 * 2)]: {
+    marginLeft: theme.spacing(3),
+    marginRight: theme.spacing(3),
+    [theme.breakpoints.up(400 + theme.spacing(3) * 2)]: {
```

Comment:
Is this the same as `400 + theme.spacing(6)` ?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

Reply:
Yes, but it's here to illustrate that we take margin left & right into account. I'm going with:
```js
[theme.breakpoints.up(400 + theme.spacing(3 * 2))]: {
```